README.md

```diff
@@ -195,6 +195,68 @@ cp sample-dcrdata.conf ~/.dcrdata/dcrdata.conf
 Then edit dcrdata.conf with your dcrd RPC settings. See the output of `dcrdata --help`
 for a list of all options and their default values.
 
+### Using Configuration Environment Variables
+There will be times when you don't want to fuss with a config file or
+cannot use command line args such as when using docker, heroku, kubernetes or other cloud
+platform.  
+
+Almost all configuation items are avaiable to set via environment variables.
+To have a look at what you can set please see config.go source code and the config struct (line 68).
```

Comment:
Sorry I missed this, but let's not refer to a line of code by number since the code will inevitably change without getting the reference updated.  How about we just have this say "the `config` struct" so that it's clear to look at the actual variable def.